docs/graphs/graph-queries/graph-keywords/shortest-path.md

```diff
@@ -0,0 +1,52 @@
+---
+title: SHORTEST_PATH
+---
+
+The `SHORTEST_PATH` keyword in [C8QL](../../../queries/c8ql/) is used to find the shortest path between two vertices in a graph. It computes the path with the fewest number of edges (hops) between a start vertex and a target vertex. If it finds two paths with the same shortest length, then it returns only one.
```

Comment:
By default the shortest path is calculated by the number of hops but a weight attribute can be specified in which case the shortest path would be the one with the smallest sum of weights over the edges in the path.

Reply:
Updated